tools/bwa.wdl

```diff
@@ -30,23 +30,29 @@ task bwa_aln {
             n_cores=$(nproc)
         fi
 
-        mkdir bwa
+        mkdir /tmp/bwa
 
-        tar -C bwa -xzf ~{bwadb_tar_gz}
-        PREFIX=$(basename bwa/*.ann ".ann")
+        tar -C /tmp/bwa -xzf ~{bwadb_tar_gz}
+        PREFIX=$(basename /tmp/bwa/*.ann ".ann")
 
-        bwa aln -t "${n_cores}" bwa/"$PREFIX" ~{fastq} > sai
+        bwa aln -t "$n_cores" /tmp/bwa/"$PREFIX" ~{fastq} > sai
```

Comment:
I don't object to this, but I do slightly prefer the `${}` syntax for bash variables inside of a WDL command block. To me, that clarifies that it is a bash variable reference and not a WDL variable.